libs/core/utils/get-serial-number-value.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+export const getSerialNumberValue = (
+  serialNumber: string | undefined
+): string => {
+  if (serialNumber === undefined) {
+    return ""
+  }
+  if (serialNumber === "00000000000000") {
```

Comment:
those zeros "00000000000000" occur few times in project, maybe make const out of it? :)